src/utils/handlers/requestHandler.ts

```diff
@@ -13,7 +13,11 @@ export const defaultContext = {
   fetch,
 }
 
-export type DefaultRequestBodyType = Record<string, any> | string | undefined
+export type DefaultRequestBodyType =
+  | Record<string, any>
+  | string
+  | any
```

Comment:
Adding `any` to the union type effectively equals just `any` type`:

```ts
type A = string | number | any
type B = any
```

Types `A` and `B` are the same when it comes to type-checking. I think we should annotate the missing body types, but not via `any`.